app/src/main/java/org/mozilla/focus/session/SessionManager.java

```diff
@@ -230,6 +230,15 @@ public void removeCurrentSession() {
         removeSession(currentSessionUUID);
     }
 
+    /**
+     * Remove the session by its UUID.
+     */
+    public void removeSessionByUUID(@NonNull int position) {
+        final Session session = this.sessions.getValue().get(position);
+        final String deleteId = session.getUUID();
+        removeSession(deleteId);
+    }
+
     @VisibleForTesting void removeSession(String uuid) {
```

Comment:
Could be renamed to `removeSessionByUUID`

Reply:
Renamed to removeSessionByUUID. 👍